mucommander-core/src/main/java/com/mucommander/ui/main/DrivePopupButton.java

```diff
@@ -221,8 +221,12 @@ private void updateButton() {
 
             }
             setText(newLabel);
-            // Set the folder icon based on the current system icons policy
-            setIcon(FileIcons.getFileIcon(currentFolder));
+            { // local ctx for icon
```

Comment:
why do we need this block?

Reply:
I didn't want to create another method scope variable for icon. If you don't like it, then I can move `icon` outside of switch clause.

Reply:
PR updated